easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.13-GCC-10.3.0.eb

```diff
@@ -0,0 +1,28 @@
+name = 'OpenBLAS'
+version = '0.3.13'
```

Comment:
OpenBLAS release 0.3.15 seems to be close, cfr. https://github.com/xianyi/OpenBLAS/milestone/34

(0.3.14 was essentially cancelled, due to regression in AVX512 SGEMM)

Reply:
Okay, I marked this PR as draft for now. I will update this as soon as 0.3.15 is released.

Reply:
Done :-)